.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "0.12"
+  - "5.0"
```

Comment:
@narqo Ehm. Not only [enterprise company title], a lot of companies, actually, will use `0.x`. But I'm not against `0.x`, I'm against sticking to `4.0`/`5.0` instead of `4.x`/`5.x`.